crates/ruff_linter/src/rules/pydocstyle/snapshots/ruff_linter__rules__pydocstyle__tests__D408_sections.py.snap

```diff
@@ -1,22 +1,16 @@
 ---
 source: crates/ruff_linter/src/rules/pydocstyle/mod.rs
 ---
-sections.py:94:5: D408 [*] Section underline should be in the line following the section's name ("Returns")
-    |
- 92 |           "section's name ('Returns')")
- 93 |   def blank_line_before_underline():  # noqa: D416
- 94 |       """Toggle the gizmo.
-    |  _____^
- 95 | | 
- 96 | |     Returns
- 97 | | 
- 98 | |     -------
- 99 | |     A value of some sort.
-100 | | 
-101 | |     """
-    | |_______^ D408
-    |
-    = help: Add underline to "Returns"
+sections.py:96:5: D408 [*] Section underline should be in the line following the section's name ("Returns")
+   |
+94 |     """Toggle the gizmo.
+95 | 
+96 |     Returns
+   |     ^^^^^^^ D408
+97 | 
+98 |     -------
```

Comment:
Again, here this would probably ideally be the range of the underline itself, rather than the range of the section header, since the lint is about the underline